src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/layer/Layergroup.java

```diff
@@ -0,0 +1,134 @@
+package de.terrestris.shogun2.model.layer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.OrderColumn;
+import javax.persistence.Table;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+
+/**
+ *
+ * Representation of a layer which consists a corresponding data source
+ * and an appearance
+ *
+ * @author Kai Volland
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+@Entity
+@Table
+public class Layergroup extends AbstractLayer {
+
```

Comment:
Yes. In the end this is a matter of taste.

I prefer the CamelCase version as it is better readable. We should also handle this consistently within the project and as we already have a model `UserGroup`, i would also like to have `LayerGroup` here :wink: